### PR TITLE
More type annotations

### PR DESCRIPTION
**Pull request by zupo**
And some general cleanup.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def run(self):  # noqa: D102
     ],
     keywords="pyramid openapi3 openapi rest restful",
     packages=find_packages(exclude=["tests"]),
-    package_data={"pyramid_openapi3": ["static/*.*"], "": ["LICENSE"]},
+    include_package_data=True,
```
Comment (zupo) [inline setup.py]: We have `graft pyramid_openapi3` in MANIFEST.IN, which means "include all files in given directory", so this should be enough.

**Comment by zupo** [comment]
@sayanarijit, @gweis: review pls if you have any free cycles.

File: pyramid_openapi3/__init__.py [hunk 2]
```diff
@@ -65,7 +65,7 @@ def openapi_view(view: View, info: ViewDeriverInfo) -> t.Optional[View]:
     """
     if info.options.get("openapi"):
 
-        def wrapper_view(context, request):
+        def wrapper_view(context: t.Any, request: Request):
```
Comment (sayanarijit) [inline pyramid_openapi3/__init__.py]: Probably this is the way:

```
ContextType = t.TypeVar('Context')
View = t.Callable[[ContextType, Request], Response]

def wrapper_view(context: ContextType, request: Request) -> Response:
```
I'll check ...

Comment (zupo) [reply pyramid_openapi3/__init__.py]: Please submit a PR if it turns out it works. It does look better!

**Pull request merged**